doc/api/stream.md

```diff
@@ -1806,6 +1812,57 @@ const myWritable = new Writable({
 });
 ```
 
+#### writable.\_construct(options, callback)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object} Options passed to the `Writable` constructor.
+* `callback` {Function} Call this function (optionally with an error
+  argument) when the stream has finished initializing.
+
+The `_construct()` method **must not** be called directly. It may be implemented
```

Comment:
```suggestion
The `_construct()` method MUST NOT be called directly. It may be implemented
```

Reply:
The other method's docs use **must not**? e.g. `_destroy`

Reply:
See https://github.com/nodejs/node/pull/29656/files#diff-94e62e3974692b345ca60e0139a9291eR1893, and I remember that other specs do use the upper-case, too.